homeassistant/components/modbus/__init__.py

```diff
@@ -204,7 +204,6 @@ def setup(self):
                 stopbits=self._config_stopbits,
                 bytesize=self._config_bytesize,
                 parity=self._config_parity,
-                timeout=self._config_timeout,
```

Comment:
Is the deletion of this line intentional?

Reply:
Yes, serial does not support timeout, and for a reason I cannot explain it is silently ignored on my installation and a few others, but I have one user where it fails.

Thanks for asking, it could have been a mistake.

Reply:
Okay thanks for the explanation. Of course ✌

Reply:
Shouldn't you then deprecate and eventually remove the config option for it?

https://github.com/home-assistant/core/blob/f2fbe657c4d7f2fc71dd4551fcc96771d3baef4a/homeassistant/components/modbus/__init__.py#L56

Reply:
No timeout is still used in the code, also for serial connections, just not passed on to pymodbus.